### PR TITLE
actually save the action on `logAction` call ;)

### PR DESCRIPTION
**Pull request by gka**
the `utils.action.logAction` function didn't actually save the action log to the DB. this PR fixes it.

### CODE REVIEW

File: utils/action.js [hunk 1]
```diff
@@ -15,5 +15,7 @@ module.exports.logAction = async function(user, key, details) {
                 ? JSON.stringify(details)
                 : details
     });
-    return action.setUser(user);
+    action.setUser(user);
```
Comment (sto3psl) [inline utils/action.js]: Is this returning a promise too and needs to be `await`ed?

Comment (gka) [reply utils/action.js]: nope. the `setFoo()` methods in a model are sync because they aren't modifying the db until the `model.save()` call

Comment (sto3psl) [reply utils/action.js]: Hmm, here they do `project.setTasks` and call `then` on it. 🤷🏻‍♂️
https://sequelize.org/master/manual/associations.html#associating-objects

Comment (gka) [reply utils/action.js]: I think that's because it's not a field but a relation

Comment (sto3psl) [reply utils/action.js]: Is it actually documented somewhere in the sequelize docs? I always have trouble finding infos about these magic setters and getters.

Comment (gka) [reply utils/action.js]: I could have also used 
```js
action.user = user;
```
but I was unsure if that would work with both `userId` and `user` model.
see https://sequelize.org/master/manual/instances.html#updating---saving---persisting-an-instance

Comment (sto3psl) [reply utils/action.js]: I see. If it works sure. I think I would have associated the user directly in `Action.create`. Something like this:
```js
await Action.create({ user_id: userId, ... })
```

File: tests/action/action.test.js [hunk 2]
```diff
@@ -12,26 +12,26 @@ test.before(async t => {
...
-    await Action.destroy({
-        where: {
-            key: 'orm-test/run'
-        }
-    });
+    // await Action.destroy({
+    //     where: {
+    //         key: 'orm-test/run'
+    //     }
+    // });
```
Comment (sto3psl) [inline tests/action/action.test.js]: I think you commented this for testing reasons.

**Pull request merged**